### PR TITLE
Add actions implementation

### PR DESCRIPTION
**Pull request by pashakondratyev**

### CODE REVIEW

File: examples/one_dot_one/src/index.ts [hunk 1]
```diff
@@ -87,6 +87,19 @@ export function go(basePath: string) {
           }],
           status: "ACTIVE",
         },
+        {
+          apiName: "create-todo",
```
Comment (pashakondratyev) [inline examples/one_dot_one/src/index.ts:91]: Action API Names are very unfortunately kebab-case :( Gateway is smart enough to handle this, but will eventually need a long term solution to migrate API Names

File: packages/legacy-client/src/client/ontology.test.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  expectFetchToBeCalledWithGet,
+  mockFetchResponse,
+} from "../util/test/fetchUtils";
+import { MockOntology } from "../util/test/mockOntology";
+import { MOCK_ORIGIN } from "../util/test/mocks/mockMetadata";
+import { mockTodoObject } from "../util/test/mocks/mockObjects";
+import { unwrapResultOrThrow } from "../util/test/resultUtils";
+import { Ontology } from ".";
+
+describe(Ontology, () => {
```
Comment (mfedderly) [inline packages/legacy-client/src/client/ontology.test.ts:32]:
```suggestion
describe("Ontology", () => {
```

File: packages/legacy-client/src/client/ontology.test.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+import {
+  expectFetchToBeCalledWithBody,
+  expectFetchToBeCalledWithGet,
+  mockFetchResponse,
+} from "../util/test/fetchUtils";
+import { MockOntology } from "../util/test/mockOntology";
+import { MOCK_ORIGIN } from "../util/test/mocks/mockMetadata";
+import { mockTodoObject } from "../util/test/mocks/mockObjects";
+import { unwrapResultOrThrow } from "../util/test/resultUtils";
+import { Ontology } from ".";
```
Comment (mfedderly) [inline packages/legacy-client/src/client/ontology.test.ts:30]: Maybe fixup this import

File: packages/legacy-client/src/client/actions.test.ts [hunk 4]
```diff
@@ -27,13 +27,11 @@ import type { Actions } from "./actions";
 import type { OsdkLegacyObjectFrom } from "./OsdkObject";
 
 describe("actions", () => {
-  it("types work", async () => {
-    const actions: Actions<
-      typeof MockOntology
-    > = {} as Actions<
-      typeof MockOntology
-    >;
+  const actions: Actions<
```
Comment (mfedderly) [inline packages/legacy-client/src/client/actions.test.ts:30]: This file was probably supposed to be ontology.test.ts

File: packages/legacy-client/src/client/ontology.test.ts [hunk 5]
```diff
@@ -0,0 +1,141 @@
+/*
```
Comment (mfedderly) [inline packages/legacy-client/src/client/ontology.test.ts:1]: This file probably should have been actions.test.ts

File: packages/legacy-client/src/client/ontology.test.ts (hunk 2 above)
Comment (pashakondratyev) [reply packages/legacy-client/src/client/ontology.test.ts:32]: I saw this used a few places, in vitest it shows the right name, don't feel super strong about this though

Comment (mfedderly) [reply packages/legacy-client/src/client/ontology.test.ts:32]: The examples from vitest use strings here, I'll add it to the backlog to move everything over to quoted strings. For some reason it accepts Functions as well.

**Review by mfedderly — APPROVED** [review]

**Pull request merged**